dotcms-integration/src/test/java/com/dotmarketing/portlets/workflows/business/WorkflowAPITest.java

```diff
@@ -4663,4 +4674,128 @@ public void test_comment() throws DotDataException, DotSecurityException {
         assertEquals("comment test 1", comments1.get(0).commentDescription());
     }
 
+    /**
```

Comment:
The test javadocs should follow this convention https://auth.dotcms.com/devwiki/integration-unit-testing-conventions

Reply:
Done